documentation/docs/sidebars.js

```diff
@@ -38,36 +38,36 @@ const sidebars = {
     },
     {
       type: 'category',
-      label: 'Atala PRISM V2',
+      label: 'Identus',
       link: {
         type: 'generated-index',
-        title: 'Atala PRISM V2',
-        description: 'Learn about the Atala PRISM ecosystem!'
+        title: 'Identus',
+        description: 'Learn about the Identus ecosystem!'
       },
       collapsed: false,
       items: [
-        'atala-prism/overview',
+        'identus/overview',
         {
           type: 'category',
-          label: 'PRISM Cloud Agent',
+          label: 'Cloud Agent',
           collapsed: true,
           link: {
             type: 'generated-index',
-            title: 'PRISM Cloud Agent',
-            description: 'Learn about the Atala PRISM Cloud Agent!'
+            title: 'Cloud Agent',
+            description: 'Learn about the Identus Cloud Agent!'
```

Comment:
Just Cloud Agent

Reply:
@mkbreuningIOHK, both terms are valid and can be used interchangeably:
- `Cloud Agent` - almost all the cases
- `Identus Cloud Agent` - to highlight or emphasize the relation to the Identus Platform 

CC: @petevielhaber